src/web/components/App.tsx

```diff
@@ -317,7 +317,10 @@ export const App = ({ CAPI, NAV }: Props) => {
                 </Lazy>
             </Portal>
             <Portal root="cmp">
-                <CMP cmpUi={CAPI.config.cmpUi} />
+                <>
+                    {/* need to assure cmpUi is false and not undefined for storybook */}
+                    {CAPI.config.cmpUi === false && <CMP />}
```

Comment:
This means the CMP component will only render if `cmpUI` is explicitly `false` - is that correct? The naming kind of suggests the opposite

Reply:
Yes, the value of `CAPI.config.cmpUi` should be `true` to show the `<CMP />`, it is derived from the `cmpUi` feature switch value, which is exposed at `window.guardian.config.switches.cmpUi` on `frontend`.

Reply:
yes, this is only rendered when `CAPI.config.cmpUi` is false, idk why this was decided, you can look in the old `CMP` and it returns null when `! cmpUi`

Reply:
Just took a look and it was previously returning `null` not the component if `!cmpUi`

Reply:
@GHaberis so this is a bug? We should change this to:
```suggestion
                    {CAPI.config.cmpUi && <CMP />}
```

Reply:
@liywjl yes your `{CAPI.config.cmpUi && <CMP />}` suggestion is correct 👍